frontend/src/scenes/instance/AsyncMigrations/AsyncMigrations.tsx

```diff
@@ -18,70 +19,51 @@ import {
+import { humanFriendlyDetailedTime } from 'lib/utils'
 
 export const scene: SceneExport = {
     component: AsyncMigrations,
 }
 
 const { TabPane } = Tabs
 
-export const tooltipMessageForStatus = {
-    0: 'Run migration',
-    1: 'Force stop migration',
-    2: 'Migration completed',
-    3: 'Re-run migration',
-    4: 'Re-run migration',
-}
-
 export function AsyncMigrations(): JSX.Element {
     const { user } = useValues(userLogic)
     const { asyncMigrations, asyncMigrationsLoading, activeTab, asyncMigrationSettings } =
         useValues(asyncMigrationsLogic)
-    const { triggerMigration, forceStopMigration, loadAsyncMigrations, setActiveTab } = useActions(asyncMigrationsLogic)
+    const {
+        triggerMigration,
+        resumeMigration,
+        rollbackMigration,
+        forceStopMigration,
+        forceStopMigrationWithoutRollback,
+        loadAsyncMigrations,
+        loadAsyncMigrationErrors,
+        setActiveTab,
+    } = useActions(asyncMigrationsLogic)
 
-    const columns = [
+    const columns: LemonTableColumns<AsyncMigration> = [
         {
-            title: '',
-            render: function RenderTriggerButton(asyncMigration: AsyncMigration): JSX.Element {
-                const status = asyncMigration.status
-                return (
-                    <Tooltip title={tooltipMessageForStatus[status]}>
-                        {status === 0 ? (
-                            <PlayCircleOutlined
-                                className="migration-btn success"
-                                onClick={() => triggerMigration(asyncMigration.id)}
-                            />
-                        ) : status === 1 ? (
-                            <StopOutlined
-                                className="migration-btn danger"
-                                onClick={() => forceStopMigration(asyncMigration.id)}
-                            />
-                        ) : status === 2 ? (
-                            <CheckCircleOutlined className="success" />
-                        ) : status === 3 || status === 4 ? (
-                            <RedoOutlined
-                                className="migration-btn warning"
-                                onClick={() => triggerMigration(asyncMigration.id)}
-                            />
-                        ) : status === 5 ? (
-                            <Spinner size="sm" />
-                        ) : null}
-                    </Tooltip>
-                )
-            },
+            title: 'id',
+            dataIndex: 'id',
         },
         {
             title: 'Migration name',
             dataIndex: 'name',
         },
+        {
+            title: 'Error cnt',
+            dataIndex: 'error_cnt',
+        },
         {
             title: 'Description',
-            render: function RenderError(asyncMigration: AsyncMigration): JSX.Element {
+            render: function Render(_, asyncMigration: AsyncMigration): JSX.Element {
```

Comment:
This column is very clunky, particularly because I have to click the tiny ellipsis button on each description just to read it to the end. Could we use a similar pattern as in Insights/Dashboards/Feature flags here, where the description is shown in the name column (class `row-description`) and not truncated?

Reply:
<img width="528" alt="Screen Shot 2022-01-24 at 15 47 44" src="https://user-images.githubusercontent.com/890921/150805158-c93bc468-2b6e-403a-ac40-791d0e657578.png">

Updated description based on the product design (https://www.figma.com/file/gQBj9YnNgD8YW4nBwCVLZf/PostHog-App?node-id=6625%3A32152). How I can make the length be tied to column length?

Reply:
not really I think the descriptions could be really long especially in the future & I think we should add more stuff there too

Reply:
also this doesn't need to be perfect yet :) I can continue to make it look nicer for next release

Reply:
ok this is what I got and it's probably fine for this PR we can improve it in future releases:
<img width="473" alt="Screen Shot 2022-01-24 at 18 22 50" src="https://user-images.githubusercontent.com/890921/150833379-a223697b-16bf-4bd8-8772-b1fca277b99d.png">